lib/main.ts

```diff
@@ -285,13 +285,16 @@ export async function viewConfession(staging_ts: string, approved: boolean): Pro
         console.log(`Updated!`);
         console.log(`Updating staging message...`);
         try {
+            const statusText = `${
+              approved ? `:true: Approved` : `:x: Rejected`
+            } by <@${reviewer_uid}> <!date^${Math.floor(Date.now() / 1000)}^{date_short_pretty} at {time}|${new Date().toISOString()}>.`;
             await web.chat.update({
                 channel: staging_channel,
                 ts: staging_ts,
                 text: '',
                 blocks: new Blocks([
                     new TextSection(new MarkdownText(`(staging) *${fields.id}* ${fields.text}`)),
-                    new TextSection(new PlainText(approved ? ":true: Approved" : ":x: Rejected")),
+                    new TextSection(new MarkdownText(statusText)),
```

Comment:
No need to change to Markdown.

Reply:
incorrect. it doesn't let you do mentions or date format without markdown. that's the fix from earlier :)